docs/developer.md

```diff
@@ -0,0 +1,64 @@
+# Mobile App Developer's Guide
+
+This guide outlines the best practices to follow when you’re building a mobile app for the Province of B.C. 
+
+
+## Digital Code of Practice
+
+All apps developed for the Province of B.C. must adhere to the [Digital Code of Practice](https://digital.gov.bc.ca/policies-standards/dcop/) (DCoP). It provides a comprehensive set of guidelines and principles to guide your work. The DCoP emphasizes user-centricity, accessibility, security, and efficiency. Adhering to it will help you achieve digital excellence and improve citizen experience.
```

Comment:
```suggestion
All apps developed for the Province of B.C. must adhere to the [Digital Code of Practice](https://digital.gov.bc.ca/policies-standards/dcop/) (DCoP). The DCoP emphasizes a user focus, accessibility, security and efficiency. Adhering to it will help you achieve digital excellence and improve the citizen experience.
```